### ISSUE TITLE
Look not being applied

### ISSUE THREAD

**Issue by travischoma** [rofrischmann/react-look#29]
Any ideas what I'm doing wrong here:

This is the portion of the component I am styling:
```
  <Layout.Center look="selectedAccount" justifyContent="space-around" row>
              <Layout.Box flex padding="10">
                <h4>{this.state.entity.companyName}
                </h4>
              </Layout.Box>
              <Layout.Box flex look="selectedAccount" padding="10">
                <MUI.RaisedButton label="Switch Account" primary={true}/>
              </Layout.Box>
            </Layout.Center>
```
My sheet:
```
let sheet = new Look({
   selectedAccount : {
    backgroundColor: 'blue'
   }
});
```
Last thing I do is applyTo
```
export default sheet.applyTo(App);
```

The component loads fine but the Look isn't applied. It seems it's just returning the plain container from the applyTo function.

**Comment by rofrischmann** [comment]
Hey @travischoma, can you tell me which version of Safari caused this issue? I have tried to reproduce your issue and it turned out to look like this: 
<img width="496" alt="screen shot 2015-07-23 at 8 11 45 am" src="https://cloud.githubusercontent.com/assets/10060928/8844209/a51c4b32-3112-11e5-9f09-9de6f5e18917.png">

**Comment by rofrischmann** [comment]
In #30 it looks like there's an issue with isCondition. This gave me an idea. I am already using Safari 9.0 as part of OS X El Capitan and they've added a lot of ES6 syntax I think. I thought Babel may transpile most.

Give me 1 hour :P

**Comment by rofrischmann** [comment]
I would love to see you testing this again. As I've a lot to do right now I tried to build this lib as fast as possible which gave me no time to write proper tests :/ I hope to add some after my vacation..

**Comment by travischoma** [comment]
The Safari used was Version 8.0.5 (10600.5.17). That was the one with the error. In Chrome Version 44.0.2403.89 (64-bit) I wasn't getting any error, but also it wasn't styling the component.

**Comment by travischoma** [comment]
Ok I retested, both Chrome and Safari are not throwing errors. But in either browser the component loads without the style applied. Any other ideas on what might be going wrong? Is there a way to turn on a debug flag so I can see at what step things are going wrong?

This issue aside, I'm looking forward to using react-look. Seems like an awesome way to handle styling elements and escape css hell :-).

**Comment by travischoma** [comment]
Ok I think I have isolated the bug. I ran the whole thing through the debugger and I see that the enhancer and the looks are being resolved in the render alright.

I can apply the style to any element no problem. However if I try to apply a style to a child nested in a Sidebar tag (The sidebar component from https://github.com/balloob/react-sidebar) it doesn't apply the style. Any ideas why applyTo is failing for children of this component? When you have time, it might be an interesting bug to look at, perhaps what ever condition is causing this to not style children of react-sidebar, could also be happening with other custom components.

**Comment by travischoma** [comment]
I came up with a work around by splitting the child content of the sidebar into a separate component and applying the styles directly to that new component and it worked.

I had one other instance where a child look wasn't being applied, and I simply made a dummy look for the parent as well and then the child's look was applied. There might be something funky going on in some situations with regards to how it walks the tree / applies styles.

**Comment by rofrischmann** [comment]
I love how you're investigating! @travischoma  I will have a look at it when I am back from vacation next week. I think i had some issue like this in the past... Looks like it does not check children of components without a look prop for some reason. 

In the meantime, if you find out what's failing exactly, be free to fix it and send a pull request :)

**Comment by travischoma** [comment]
Fixed it with pull request #33 :-) Enjoy your vacation!
